### ISSUE TITLE
Dev: fix Alert: Orphaned form label on Archived Accounts page

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/tdm-calculator#3039]
### Overview
We need to fix the Alert: Orphaned form label on Archived Accounts so that page can be WCAG compliant.

#### Details
Category: Alert
Name: Orphaned form label
(The Accessibility wiki for this Alert has a solution)

Page that will be worked on:
Archived Accounts

### Action Items
- [ ] Review wiki page: Accessibility, How to run the WAVE browser extension (Resource 1.01)
   - [ ] Install the WAVE browser extension
- [ ] Log in as the user indicated (resource 1.02.01) 
- [ ] navigate to the page you are going to fix (resource 1.02.02)
- [ ] Open the TDM WAVE Accessibility Audit Spreadsheet, for this Alert (resource 1.03) and find the row that relates to this  issues page.
- [ ] Run the WAVE extension with no elements focused, or clicked
- [ ] Find the Alert referenced in the Spreadsheet (1.03)
- [ ] This alert has been resolved on other pages, so review `Project Team Solution` section (resource 1.04.01.01)
- [ ] Apply solution
- [ ] Submit PR
- [ ] Add PR to spreadsheet in PR column next to Alert (resource 1.03)
- [ ] On this issue
   - [ ] Add labels: 
      - `ready for accessibility lead`
   - [ ] Change status to: `Technical Review`
   - to the epic #2896
     - [ ] Add the label `ready for product`
      - [ ] Add this comment 
         ```
          - #3039 is closed.  Please review to see if there are any remaining instances of this problem
         ```

### Resources/Instructions
- 1.01[How to run the WAVE browser extension](https://github.com/hackforla/accessibility/blob/main/docs/getting-started/wave-browser-extension.md)
- 1.02 Application details
   - 1.02.01 User Profile Required: Security
   - 1.02.02 Page URL: https://tdm-dev.azurewebsites.net/archivedaccounts
      - Requires sign in: TRUE
      - Can access page directly from URL: FALSE
- 1.03 [TDM WAVE Accessibility Audit Spreadsheet, tab: 2. Audit Log, filter: Alert: Orphaned form label](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0&fvid=336057419)
   -  Spreadsheet Row(s): 51
- 1.04 [Accessibility Known Issues Index](https://github.com/hackforla/accessibility/blob/main/docs/known-issues-and-solutions/solutions-index/index.md)
    - 1.04.01 [WAVE Known Accessibility Issue: Alert - Orphaned form label](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/alert-orphaned-form-label)
        - 1.04.01.01 [WAVE Known Accessibility Issue: Alert - Orphaned form label, section: ProjectTeam Solution](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/alert-orphaned-form-label/#project-team-solution)

Additional reading (optional)
- https://hackforla.github.io/accessibility/

#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - [Staging Deck, slide 3039](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g39673b86911_4_21#slide=id.g39673b86911_4_21)
- Release deck
   - [2026-MM-DD Release deck, slide 3039]

**Comment by Rabia2219** [comment]
@geolunalg 
Please add a level label. You can choose from one of the following:

- `good first issue`
- `level: easy`
- `level: medium`
- `level: hard`

Once you have added the level label, please:
- remove the `level: missing` label
- remove the `ready for accessibility lead` label
- add the `ready for prioritization` label

**Comment by Philc90** [comment]
Verified this WAVE Alert no longer appears on the Archived Accounts page, appears to have been fixed in previous run.

<img width="1798" height="879" alt="Image" src="https://github.com/user-attachments/assets/bd83f594-922f-4023-b8f1-f32f758f2fc7" />

**Comment by geolunalg** [comment]
This was addressed in the first round of accessability and no PR made.
`- #3039 is closed.  Please review to see if there are any remaining instances of this problem
`

**Comment by ExperimentsInHonesty** [comment]
@geolunalg @Philc90 
- I updated the spreadsheet, and am closing this issue. The fix was made on issue #2719.  Contrary to instructions below do not expand scope of this issue

>- #3039 is closed.  Please review to see if there are any remaining instances of this problem
